website/sidebars.ts

```diff
@@ -221,6 +221,44 @@ export default {
       'systrace',
       'transforms',
       'vibration',
+      {
+        type: 'category',
+        label: 'Globals',
+        collapsed: false,
```

Comment:
```suggestion
        collapsed: true,
```

Due to shear amount of the pages, we might want to collapse the section by default, to avoid large scrollable sidebar initially. We might want to collapse also further groups, to match what we do in "Guides".

Alternatively, if we want to retain visibling, we could move the "Globals" section at the end of sidebar.